pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget.go

```diff
@@ -477,7 +489,16 @@ func (c *DisruptionBudgetController) createPDB(key string, vmi *virtv1.VirtualMa
 				},
 			},
 		},
-	}, v1.CreateOptions{})
+	}
+
+	if c.onOpenShift {
+		pdb.Labels = map[string]string{
+			OpenShiftPDBAtLimitAlert: "disabled",
+		}
+	}
+
+	c.podDisruptionBudgetExpectations.ExpectCreations(key, 1)
+	createdPDB, err := c.clientset.PolicyV1beta1().PodDisruptionBudgets(vmi.Namespace).Create(context.Background(), pdb, v1.CreateOptions{})
```

Comment:
i just caught that this label is only added during creation. If you want this to occur during updates for existing VMIs, you'll need to add the label to the patch logic that expands the PDB as well.

Reply:
Good catch! Didn't think about that, I've changed it now but how should we test this aspect? It would require a downgrade/upgrade e2e test, should I add that?

Reply:
should be easy. Just mutate one of the pdb's in your existing functional test to remove the label and make sure it magically gets re-added

Reply:
Done, thanks for the suggestion, can you take another look?